src/strategies/izumi-veizi/index.ts

```diff
@@ -0,0 +1,79 @@
+import { BigNumberish } from '@ethersproject/bignumber';
+import { formatUnits } from '@ethersproject/units';
+import { Multicaller } from '../../utils';
+import abi from './abi.json'
```

Comment:
It is not required to pass the entire json, just pass the abi of the method you are using :)